packages/charts-vanilla-radar/src/index.js

```diff
@@ -0,0 +1,241 @@
+    });
+
+    this.axis.names = this.data[0].map( el => el.axis);
+    this.axis.length = this.axis.names.length;
+
+    //puts 0,0 in the center of the svg
+
+    // draws the axis
+    if (this.cfg.axisStyle) {
+      this.axisGroup = globalGroup.append('g').attr('class', 'axis-lines');
+      this.axisGroup.selectAll('.axis-line').data(this.axis.names).enter().append('line')
+        .attrs({
+          'x1': 0,
+          'y1': 0,
+          // `+ Math.PI/2 to make the Axis line up with the correct Data`
+          'x2': (d, i) => {
+            return this.scale(this.cfg.max * 1.05) * Math.sin( this.cfg.angleSlice * i + Math.PI / 2);
+          },
+          'y2': (d, i) => {
+            return this.scale(this.cfg.max * 1.05) * Math.cos( this.cfg.angleSlice * i + Math.PI / 2);
+          },
+          'opacity': 0.3,
+          'class': 'axis-line',
+        }).styles(this.cfg.axisStyle);
+    }
+
+    if (this.cfg.shape == 'circle') {
+      //draw rings
+      const ringInterval = this.cfg.max / this.cfg.levels;
+      //add 1 so you include the MAX
+      const ringRadiusValues = range(ringInterval, this.cfg.max + 1, ringInterval).reverse();
+
+      let g = globalGroup.append('g').attr('class', 'grid-circle');
+      g.append('g').attr('class', 'rings')
+        .selectAll('.ring').data(ringRadiusValues)
+        .enter().append('circle').attrs({
+          'fill': 'none',
+          'cx': 0,
+          'cy': 0,
+          'r': (d) => { return this.scale(d); },
+          'stroke': 'black',
+          'stroke-width': 1,
+          'stroke-dasharray': '5 5',
+          'opacity': 0.2,
+          'class': 'ring',
+        });
+
+      //add the unit labels on the rings
+      g.append('g').attr('class', 'rings-labels')
+        .selectAll('.ring-label').data(ringRadiusValues)
+        .enter().append('text').attrs({
+          'class': 'ring-label',
+          'x': '0',
+          'y': (d) =>  {return  -this.scale(d);},
+          'dy': '-0.3em',
+          'dx': '0.5em',
+          'font-size': '0.8em',
+          'fill': 'grey',
+          'opacity': 0.4,
+        })
+        .text((d) => {return d + this.cfg.units;});
+      g.append('g').attr('class', 'legend')
+        .selectAll('.axis-label').data(this.data[0]).enter().append('text').attrs({
+          'class': 'axis-label',
+          'x': (d, i) => {
+            return (this.scale(this.cfg.max) * 1.05 + this.cfg.labelFactor ) *
+              Math.sin(this.cfg.angleSlice * i + Math.PI / 2);
+          },
+          'y': (d, i) => {
+            return (this.scale(this.cfg.max) * 1.05 + this.cfg.labelFactor ) *
+              Math.cos(this.cfg.angleSlice * i + Math.PI / 2);
+          },
+          'font-size': '0.8em',
+          'fill': 'grey',
+          'opacity': 0.4,
+          'text-anchor': 'middle',
+        })
+        .text((d) => {return d.axis;})
+        .call(this.wrap, this.cfg.wrapWidth);
+
+
+
+
+    } else if (this.cfg.shape == 'polygon') {
+      //TODO
```

Comment:
@seejamescode if you've looked at the code, and are happy with it then it should be merged. I created stories for these to do's, and we can tackle them after this pull request goes in.